src/AppBundle/Command/AddUserCommand.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace AppBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use AppBundle\Entity\User;
+
+/**
+ * Creates users and stores them in the database
+ *
+ * @author Javier Eguiluz <javier.eguiluz@gmail.com>
+ */
+class AddUserCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('app:add-user')
+            ->setDescription('Creates users and stores them in the database')
+            ->addArgument('username', InputArgument::REQUIRED, 'The username of the new user')
+            ->addArgument('password', InputArgument::REQUIRED, 'The plain password of the new user')
+            ->addArgument('email', InputArgument::REQUIRED, 'The email of the new user')
+            ->addOption('is-admin', null, InputOption::VALUE_NONE, 'If set, the user is created as an administrator')
+        ;
+    }
+
+    protected function initialize(InputInterface $input, OutputInterface $output)
+    {
+        $this->em = $this->getContainer()->get('doctrine')->getManager();
+
+        $this->username = $input->getArgument('username');
+        $this->plainPassword = $input->getArgument('password');
+        $this->email = $input->getArgument('email');
+        $this->isAdmin = $input->getOption('is-admin');
```

Comment:
And I don't see the advantage of initializing the input to properties - we can just get the values right inside `execute()` :).

Also (feature creep!) it'd be cool to show off interact by asking them these questions.